lms/static/sass/views/_verification.scss

```diff
@@ -1143,9 +1144,8 @@
 
       .help-item {
         @include fill-parent;
-
-        float: left;
-        margin-right: flex-gutter();
+        // @include float(left);
```

Comment:
If they aren't needed anymore, you can just remove these rules instead of commenting them out.